src/Tooltip.tsx

```diff
@@ -124,7 +124,7 @@ const Tooltip = (props: TooltipProps, ref) => {
       mouseEnterDelay={mouseEnterDelay}
       {...extraProps}
     >
-      {children}
+      <>{children}</>
```

Comment:
This isn't an optimal solution but appears to be part of an older issue. I discovered this solution while reviewing a similar issue in [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18051).

Reply:
Why need `Fragment`?

Reply:
The issue from DefinitelyTyped that I posted above suggested wrapping `children` in an element. However, as it has been quite a while since I raised this PR I'm afraid I do not remember much else. I am going to remove it simply because it doesn't seem necessary.